### PR TITLE
Secrets Handling + Google Maps Integration

### LINKED ISSUE #3: Shift Application secrets + Configs to env
Shift secrets and url configs to env variables 
Create a seperate class handler for the same.

### PR DESCRIPTION
**Pull request by KoruptTinker**

### CODE REVIEW

File: publish/views.py [hunk 1]
```diff
@@ -105,17 +116,29 @@ def create_route(request):
...
         attach_user_to_route(request.session['username'], route['_id'])
+        if(request.POST.get("slat")):
+            route["s_lat"] = request.POST.get("slat")
+            route["s_long"] = request.POST.get("slong")
+        if(request.POST.get("dlat")):
+            route["d_lat"] = request.POST.get("dlat")
+            route["d_long"] = request.POST.get("dlong")
+
+        if(request.POST.get("dlat") and request.POST.get("slat")):
+            res = mapsService.get_route_details(route["s_lat"], route["s_long"], route["d_lat"], route["d_long"])
```
Comment (Abhi0010) [inline publish/views.py:138]: Add error handling if the get_route_details() API call cannot find a route between the source and destination.

Comment (KoruptTinker) [reply publish/views.py:138]: Added error handling !

Comment (Abhi0010) [reply publish/views.py:138]: Add a timeout for the request

Comment (KoruptTinker) [reply publish/views.py:138]: Added!

**Pull request merged**